pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -185,17 +184,20 @@ def __init__(
 
         self.kth_value, self.mode = mode_dict[mode]
 
+    @rank_zero_only
     def _del_model(self, filepath: str):
         if self._fs.exists(filepath):
             self._fs.rm(filepath)
+            rank_zero_info(f"Removed checkpoint: {filepath}")
```

Comment:
just info is fine as all method is rank zero only

Reply:
here i prefer rank_zero_info so we can remove the `from pytorch_lightning import _logger as log` import altogether